### ISSUE TITLE
login error with upgraded Frame

### ISSUE THREAD

**Issue by hydrotik** [hydrotik/node-hapi-react-redux-sass-typescript-mongo-webpack-hmr-gulp#112]
```
[3]             request.auth.session.set(result);
[3]                                 ^
[3] 
[3] TypeError: Cannot read property 'set' of undefined
```

**Comment by hydrotik** [comment]
60888ecc1275b8feb0978dd64be15c2bd6ff43ae

**Comment by hydrotik** [comment]
@dhsaks Can you take a look at this? I also think this might have solved some other issues once it is working correctly

https://github.com/hydrotik/node-hapi-react-redux-sass-typescript-mongo-webpack-hmr-gulp/issues/108

**Comment by hydrotik** [comment]
https://github.com/jedireza/aqua/issues/85

**Comment by hydrotik** [comment]
Check this PR and maybe do a setup of the project and see if that error shows up:
https://github.com/jedireza/aqua/tree/update-deps

**Comment by hydrotik** [comment]
So there is a discrepancy between the auth strategy between Frame and the aqua reps PR. Currently this is using the code in the update-deps branch and not in Frame.

In getting that branch working I did upgrade crumb to 6 and hapi to 13 which it was not, which caused an error in the cookie secret length. I updated this to a 32 char hash and it worked. I moved it into the Watts glue upgrade branch and it still didn't fix the issue. There seems to be some missing component which is why the session object isn't registering. Still probing.
